content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Accessing HPE Ezmeral Data Fabric Object Storage from Spring Boot S3
+  Micro Service deployed in K3s cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: false
+priority: 0
+author: Kiran Kumar Mavatoor
+authorimage: https://gravatar.com/avatar/dddba13c1f3712cb7d1ae9a1111d86d6?s=96
+tags:
+  - K3S
+  - Spring Boot
+  - HPE Ezmeral Data Fabric Object Storage
+  - hpe-ezmeral-data-fabric
+  - MapR
+  - Quarkus
+  - Kubernetes
+  - S3
+  - Object Storage
+  - IOT
+---
+Containers and Microservices are transforming Edge and IoT platform use cases which can be deployed in Kubernetes clusters on edge nodes with small footprint and fast processing by persisting data at central location. This data pipeline can be easily accessed by downstream complex analytics applications for further processing. 
+
+In this article, I will discuss how to access HPE Ezmeral Data Fabric Object Store (S3) using Spring Boot S3 Micro Service application deployed in a [K3s cluster](https://k3s.io/) and perform basic S3 operations like upload, list, delete etc. The below diagram gives an overview of the architecture.
+
+![](/img/hpe-ezmeral-data-fabric-s3-springboot-k3s.png)
+
+Figure 1:  Architecture overview of Spring Boot S3 Micro Service on K3s with HPE Data Fabric Object Storage as the back end.
```

Comment:
Recommend changing this caption "Figure 1: Architecture overview of SpringBoot S3 Micro Service on K3s with HPE Data Fabric Object Storage as the back end." to a **smaller font**.

Reply:
How to change the font size?. I am not able to see the option in Richtext mode